tower-http/src/services/fs/serve_dir.rs

```diff
@@ -347,6 +375,16 @@ impl<ReqBody> Service<Request<ReqBody>> for ServeDir {
     }
 }
 
+fn try_parse_range(
+    maybe_range_ref: Option<&String>,
```

Comment:
&str instead?

Reply:
Won't compile, takes an `Option<String>::as_ref() -> Option<&String>`

Reply:
You can use [`Option::as_deref`](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.as_deref)

Reply:
These kinds of small refactorings we can always make. I don't think this should be blocking.